subcommands/targets/image.go

```diff
@@ -19,15 +23,28 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 	}
 	cmd.AddCommand(imageCmd)
+	imageCmd.Flags().StringVarP(&appsShortlist, "apps", "", "",
+		"comma,separate,list of Target apps to preload into a resultant image")
```

Comment:
I suppose we should say the default is "all apps"

Reply:
+